### ISSUE TITLE
[Bug]: Any operation on matplotlib fails due to gtk4 backend error

### ISSUE THREAD

**Issue by IonizedPro45** [matplotlib/matplotlib#25602]
### Bug summary

This may be an Arch Linux dependency issue, but when using GTK4 as the backend any operation that involves a window will crash the Python program. 

### Code for reproduction

```python
import matplotlib.pyplot as plt
fig, ax = plt.subplots()
```

### Actual outcome
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mikayah/.local/lib/python3.10/site-packages/matplotlib/pyplot.py", line 1450, in subplots
    fig = figure(**fig_kw)
  File "/home/mikayah/.local/lib/python3.10/site-packages/matplotlib/_api/deprecation.py", line 454, in wrapper
    return func(*args, **kwargs)
  File "/home/mikayah/.local/lib/python3.10/site-packages/matplotlib/pyplot.py", line 783, in figure
    manager = new_figure_manager(
  File "/home/mikayah/.local/lib/python3.10/site-packages/matplotlib/pyplot.py", line 359, in new_figure_manager
    return _get_backend_mod().new_figure_manager(*args, **kwargs)
  File "/home/mikayah/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 3513, in new_figure_manager
    return cls.new_figure_manager_given_figure(num, fig)
  File "/home/mikayah/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 3518, in new_figure_manager_given_figure
    return cls.FigureCanvas.new_manager(figure, num)
  File "/home/mikayah/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 1703, in new_manager
    return cls.manager_class.create_with_canvas(cls, figure, num)
  File "/home/mikayah/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 2830, in create_with_canvas
    return cls(canvas_class(figure), num)
  File "/home/mikayah/.local/lib/python3.10/site-packages/matplotlib/backends/backend_gtk4.py", line 79, in __init__
    **css.load_from_data(b".matplotlib-canvas { background-color: white; }")
TypeError: Gtk.CssProvider.load_from_data() takes exactly 3 arguments (2 given)**
```

### Expected outcome

Not crashing

### Operating system

Arch

### Matplotlib Version

3.6.3

### Matplotlib Backend

GTK4Agg

### Python version

3.10.10

### Installation

Linux package manager

**Comment by oscargus** [comment]
One the other hand there was a fix in https://github.com/matplotlib/matplotlib/pull/25280 that seems very related. This was merged into 3.7.1, so if you get hold of the latest Matplotlib version, it should work!(?)

https://github.com/matplotlib/matplotlib/blob/5ce21e3fe930f0850eb9c4b09441aa968e24487f/lib/matplotlib/backends/backend_gtk4.py#L78-L81

**Comment by ksunden** [comment]
Given this does appear to be addressed on `main` (and even a release) already, as @oscargus points out, I'll go ahead and close this, feel free to re-ping us/reopen if that does not actually address the issue.
